src/backend/src/modules/configuration/configuration.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import * as fs from 'fs';
+@Injectable()
+export class ConfigurationService {
+  getFeatures() {
+    try {
+      // TODO need to check how to get the path of config file
+      // hardcode in here or get from a env variables
+      const filePath = process.env.FEATURE_FLAG_FILE_PATH;
+      const fileName = process.env.FEATURE_FLAG_FILE_NAME;
+      const text = fs.readFileSync(filePath + fileName, 'utf8');
+      const jsonData = JSON.parse(text);
```

Comment:
Please rename the variables as they hold the data. Calling them `text` and `jsonData` does not look like a good naming strategy.